tailcfg/tailcfg.go

```diff
@@ -475,6 +475,14 @@ type Hostinfo struct {
 	//       require changes to Hostinfo.Equal.
 }
 
+// SSHEnabled returns whether or not this node is acting as a
+// Tailscale SSH server.
+func (hi *Hostinfo) SSHEnabled() bool {
+	// Currently, we use `SSH_HostKeys` as a proxy, but we may later rely on a
+	// separate flag.
```

Comment:
mention that in the future SSH_HostKeys might include non-Tailscale SSH host keys?